api/config/v1alpha1/helpers.go

```diff
@@ -9,6 +9,7 @@ import (
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/pointer"
```

Comment:
so many pointer libs, we have one more in `internal/gatewayapi`

Reply:
"k8s.io/utils/pointer"

we use this pkg in multiple places